autoload/zsh-learn-Editl

```diff
@@ -0,0 +1,26 @@
+# -*- mode: sh -*-
+# vim: set ft=sh:
+function zsh-learn-Editl(){
+    if [[ -z $1 ]]; then
+        echo "Usage: zsh-learn-Edit <id>"
+        return 1
+    fi
+
+    content=$(echo "SELECT learning FROM $ZPWR_SCHEMA_NAME.$ZPWR_TABLE_NAME WHERE id = $1;" | ${=ZPWR_LEARN_COMMAND} --silent)
```

Comment:
The `--silent` option at the end here removes the column names from the output. This assumes the learn command is some form of `mysql`. I think that's the only database supported currently? If others are supported might need to rethink this part.

Reply:
yeah, pipe into `tail +2` instead